src/ps/handlers/interface.ts

```diff
@@ -26,11 +26,12 @@ export function interfaceHandler(message: PSMessage) {
 			if (!game) return; // Don't put any errors here! People should be able to close games that don't exist, like ones that ended
 
 			const user = message.author.id;
-			if (game.hasPlayer(user)) {
+			const player = Object.values(game.players).find(p => p.id === user);
```

Comment:
Could you check whether `game.getPlayer` works for this? It should do pretty much the same thing